blockio-uring.cabal

```diff
@@ -17,22 +17,27 @@ description:         Support for disk I/O operations using the Linux io_uring
                      it uses the io_uring kernel API. It only supports disk
                      operations, not socket operations.
 
--- bug-reports:
 license:             MIT
 license-file:        LICENSE
 author:              Duncan Coutts
 maintainer:          duncan@well-typed.com
-copyright:           (c) Well-Typed LLP 2022
+copyright:           (c) Well-Typed LLP 2022 - 2024
 category:            System
 build-type:          Simple
-extra-source-files:  CHANGELOG.md
+extra-doc-files:     CHANGELOG.md
+                     README.md
+
+source-repository head
+  type:     git
+  location: https://github.com/well-typed/blockio-uring
 
 library
   exposed-modules:     System.IO.BlockIO
   other-modules:       System.IO.BlockIO.URing
                        System.IO.BlockIO.URingFFI
-  -- other-extensions:
-  build-depends:       base, array, unix
+  build-depends:       base >= 4.16 && < 4.20
+                     , array ^>= 0.5
+                     , unix ^>= 2.8
```

Comment:
If we want to support earlier versions of `unix`, then we'll have to add some CPP pragmas to each use of `openFd`, since that has seen a breaking change with the change to `unix-2.8.0.0`